curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-regular-expressions-by-building-a-spam-filter/64220f22dff8151f751a53a7.md

```diff
@@ -0,0 +1,177 @@
+---
+id: 64220f22dff8151f751a53a7
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+A <dfn>capture group</dfn> allows you to define a part of your expression that should be captured and saved for later reference. Capture groups are defined by wrapping a part of your expression in parentheses. For example, `/h(i|ey) camper/` would match either `hi camper` or `hey camper`, and would capture `i` or `ey` in a group.
+
+Turn your place values into a capture group.
```

Comment:
This is mostly on me, but I was a bit confused where the capture group should be added. I ended up adding it to `helpRegex` as well. It might be overkill, but perhaps one test could check if `helpRegex` is unchanged.